### PR TITLE
Feat/dynamic commands and /next command

### PR DESCRIPTION
**Pull request by vcabrales**
### What does this PR do?
Logic to add commands to bot has been modified to just create and export your functions in commands js file and they will be automatically added to the bot after the services is restarted.
Also the next command has been implemented, eg: /next
/0fucks command changed to /zerofucks, function names can't start with digits
### How do I test it?
 npm install 
 npm run start:dev  and can send the commands mentioned in the PR
### Background context
When we want to add a new command to the bot, the command had to be added manually after its definition, now you just create the function.
When someone has selected the place to have lunch, the turn should change to the next person

### CODE REVIEW

**Comment by ghost** [comment]
LGTM but we are using 2 spaces instead of 4

**Comment by ghost** [comment]
you can run `npm run lint` to find what you need to fix

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+dist/
```
Comment (ghost) [inline .eslintignore]: Not sure if we need to add node_modules/ 
cc @bokvilla

File: app/commands.js [hunk 2]
```diff
@@ -3,96 +3,131 @@ import staticResponses from './data/staticResponses.js';
...
-  
-  app.command('chupala', ctx => {
-    const index = getRandomNum(staticResponses.chupala.length);
-    ctx.reply(staticResponses.chupala[index]);
+export const next = (ctx) => {
+  const people = cache.getValue('people');
+  findCurrentMorto().then((morto) => {
+    removeCurrent(morto.name);
+    return morto.name;
+  }).then((result) => {
```
Comment (ghost) [inline app/commands.js]: instead of `(result)` you could rename to `(mortoName)` and avoid extra variables.

**Review by jorgetinoco — APPROVED** [review]

**Review by ghost — APPROVED** [review]
👍

**Pull request merged**